src/main/java/uk/gov/hmcts/reform/blobrouter/tasks/utils/LoggingHelper.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.reform.blobrouter.tasks.utils;
+
+import org.slf4j.Logger;
+
+public final class LoggingHelper {
```

Comment:
this is also executing the tasks ? so probably `ScheduledTaskRunner` ?

Reply:
it's not running scheduled tasks.
Scheduled tasks are run automagically by Spring, then scheduled tasks use this class to surround code with logs.

Reply:
Or maybe `JobRunner`?

Reply:
perhaps worth considering continuously developing/improving/bringing-up-to-speed app insights setup for the service by making #204 pass. Currently it's reporting that rejected tasks are not working properly. Same thing is also done in send letter service too. Then logging will be more trivial than ever. and will not involve in tampering schedules themselves

btw, @nitinprabhu, what you are referring perhaps closely looks like the #102, which is, again, setup for #204 thing to happen

Reply:
`JobRunner` sounds good @lgonczar  @rkondratowicz